packages/grafana-ui/src/utils/flotPairs.ts

```diff
@@ -1,16 +1,14 @@
 // Types
-import { NullValueMode, DataFrame, GraphSeriesValue } from '@grafana/data';
+import { NullValueMode, GraphSeriesValue } from '@grafana/data';
 
 export interface FlotPairsOptions {
-  series: DataFrame;
+  rows: any[][];
```

Comment:
Changed getFlotPairs so it takes rows instead of series because we internally only used rows anyway. Didn't want to create a DataFrame from already parsed GraphSeriesXY